fe/fe-core/src/main/java/org/apache/doris/planner/RuntimeFilterGenerator.java

```diff
@@ -122,6 +122,24 @@ private RuntimeFilterGenerator(Analyzer analyzer) {
         bloomFilterSizeLimits = new FilterSizeLimits(sessionVariable);
     }
 
+    static class HasConjunctPredicate implements com.google.common.base.Predicate<PlanNode> {
+        @Override
+        public boolean apply(PlanNode arg) {
+            // for simplicity, skip join node( which contains more than 1 tuple id )
+            return arg.getTupleIds().size() == 1 && !arg.conjuncts.isEmpty();
```

Comment:
Returns the tuple id that satisfies the condition.

Reply:
Done